priv/gettext/ru/LC_MESSAGES/default.po

```diff
@@ -1,2 +1,19 @@
+# Duty notification text
 msgid "You are on duty tomorrow!"
-msgstr "Завтра ты назначен ответственным за лопату!"
+msgstr "Завтра ты назначен ответственным за лопату (дежурным)!"
```

Comment:
I think it would be nice to clarify this, as it can be incomprehensible for the first time for user

Reply:
Agree